cms/settings/base.py

```diff
@@ -972,6 +972,7 @@
 # API bases
 ONS_API_BASE_URL = env.get("ONS_API_BASE_URL", "https://api.beta.ons.gov.uk/v1")
 DATASETS_API_BASE_URL = env.get("DATASETS_API_BASE_URL", f"{ONS_API_BASE_URL}/datasets")  # used for dataset choosers
+DATASET_EDITIONS_API_URL = env.get("DATASET_EDITIONS_API_URL", f"{ONS_API_BASE_URL}/dataset-editions")
```

Comment:
Nit: As it is not a separate API service but another endpoint of the dataset API, I suggest naming it `DATASETS_API_EDITIONS_URL`.

I guess our usage above is incorrect, it is not longer `DATASETS_API_BASE_URL`, but more `DATASETS_API_DATASETS_URL`